### ISSUE TITLE
Invalid Java Home

### ISSUE THREAD

**Issue by stefanoportelli** [forcedotcom/sfdx-scanner#316]
I tried to launch the script but received the following message:

ERROR running scanner:run:  Invalid Java Home: C:\Program Files\Java\jdk1.8.0_231. [Todo: Instructions on how to setup correctly]

On the Prerequisites section there aren't any suggestion about that.
My current configuration is the following:
C:\>echo %JAVA_HOME%
C:\Program Files\Java\jdk1.8.0_231

Thanks in advance,
Stefano

**Comment by rmohan20** [comment]
Hi @stefanoportelli - could you check 
1. if the path (`C:\Program Files\Java\jdk1.8.0_231`) really exists?
2. if the path exists, is there a `bin\java` under it?
3. if your installed Java version is indeed 1.8? (By running `<your_path>\bin\java --version`)

If you find that all of the above is true in your case, can you please update us with the output of `<your_path>\bin\java --version`? We can check if this is a parsing issue.

Also, thanks for pasting us the message - will create a new bug for fixing the error message to include helpful information.

**Comment by stefanoportelli** [comment]
Hi @rmohan20 and thanks for you reply!
Double checked and seems a noob problem ^_^''
The JAVA_HOME variable was pointing to a non existent folder. Redirecting to the right one (..\jre1.8.0_271 instead of ..\jdk1.8.0_231) now works!

**Comment by rmohan20** [comment]
@stefanoportelli Awesome! Thanks for the update.

I'll leave the issue open so that we get to fix the error message.

**Comment by ranekere-sfdc** [comment]
This issue is fixed as part of Scanner 2.5 release.

**Comment by AdvoLantern** [comment]
Pretty old issue. But we tried this on newer FSDX Scanner plugin v3.17.0 today and have got following error:

```
ERROR running scanner:run:  The Java Home is invalid: C:\Program Files\Java\jdk-17.0.5. Error code: ENOENT.
Please verify that Java 1.8 or later is installed on your machine and try again.
If the problem persists, please manually add a 'javaHome' property to your Config.json file, referencing your Java home directory.
```

Where do we set this path?

From Environment Variables, we do have JAVA_HOME  set to this:
> echo %JAVA_HOME%
C:\Program Files\Java\jdk-17
